src/trackedit/internal/trackeditactionscontroller.cpp

```diff
@@ -336,6 +336,9 @@ void TrackeditActionsController::clipDelete(const ActionData& args)
         return;
     }
 
+    //! TODO AU4: improve for multiclip selection
+    selectionController()->resetSelectedClips();
```

Comment:
let's do this inside selectionController, I see we're already updating the state for the tracks there, see `Au3SelectionController::init`

Reply:
I'd opt for leaving it as is, because:
1. we're using selectionController this way in a few more functions here - and it seems ok to me btw, it's easily readable/understandable that some action is clearing the selection
2. In this particular function we're getting clipKey as an argument -> we'll be able to easily modify multi-selection by simply removing this clipKey from selectedClips vector. If we move it to `Au3SelectionController::init` -> we'll  rely on `trackChanged()` notification -> hence we'll need to loop over every clip to find the one that was removed
3. Not every `trackChanged` should result in resetting selection - so that may cause writing a lot of additional logic in `init()` func

wdyt?

Reply:
ok